doc/changelog/07-vernac-commands-and-options/15116-make-infoH-a-command.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  The ProofGeneral specific tactical infoH was turned into
+  a vernacular command
```

Comment:
IMHO this changelog is super obscure for users, maybe describing what the impact it should be more appropiate?

Reply:
`infoH` is only used internally by PG, I am not sure it's worth documenting further the thing.

Reply:
I just mean that the current message is ininteligible by users, removing it is better.

Reply:
IMHO

Reply:
ah, unintelligible